public/templates/public/blocks/staffpage_helppanel.html

```diff
@@ -0,0 +1,11 @@
+<fieldset>
+    {% if self.heading %}
+        <legend>{{ self.heading }}</legend>
+    {% endif %}
+    <div class="{{ self.classname }}">
+    	<h2>Want to edit your staff page?</h2>
+    	<h3>Edit your info on your <a href="/admin/pages/{{ self.instance.get_staff_page_id }}/edit/">Loop staff page</a></h3>
```

Comment:
This makes sense. Even simpler than what I was thinking. I guess I was thinking we needed to use the `Site` object to get the first part of the url for Loop. But when I see this and think about it more, I can see that the user is already going to be authenticated no matter what and it doesn't matter what URL they came in on :+1:

Reply:
Interesting to see `self.instance...`. I don't believe we do that anywhere else in the site and I didn't know we could. It makes sense though, especially since we're in the admin view.